packages/cloud-cognitive/src/components/CreateModal/CreateModal.test.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { TextInput } from 'carbon-components-react';
+
+import { pkg } from '../../settings';
+import '../../utils/enable-all'; // must come before component is imported (directly or indirectly)
+
+import uuidv4 from '../../global/js/utils/uuidv4';
+
+const blockClass = `${pkg.prefix}--create-modal`;
+
+import { CreateModal } from '.';
+const componentName = CreateModal.displayName;
+
+const className = `class-${uuidv4()}`;
+const title = 'This is a test title';
+const subtitle = 'This is a test subtitle';
+const description =
+  'This is a test description. It has several lines. It should render a modal.';
+const primaryFocus = '.bx--text-input';
+
+// render a CreateModal with title, subtitle, description, and any other required props
+const renderComponent = ({ ...rest }) =>
+  render(
+    <CreateModal
+      open
+      {...{
+        title,
+        subtitle,
+        description,
+        primaryFocus,
+        disableSubmit: false,
+        ...rest,
+      }}>
+      <TextInput
+        key="form-field-1"
+        id="1"
+        labelText="Text input label"
+        helperText="Helper text goes here"
+        placeholder="Placeholder"
+      />
+    </CreateModal>
+  );
+
+describe(componentName, () => {
```

Comment:
This we were just talking about testing guidelines, I noticed you don't have a test that `additional HTML attributes are passed through to an appropriate node`. See [here](https://github.com/carbon-design-system/ibm-cloud-cognitive/blob/main/.github/CODE_GUIDELINES.md) for details and an example.